chapter_08_events_and_message_bus.asciidoc

```diff
@@ -458,14 +458,19 @@ def allocate(
         if product is None:
             raise InvalidSku(f'Invalid sku {line.sku}')
         batchref = product.allocate(line)
-        uow.commit()
+        uow.commit() #<1>
 
         if batchref is None:
             messagebus.handle(events.OutOfStock(line.sku))
         return batchref
 ----
 ====
 
+<1> We commit even if we fail to allocate because the code is simpler this way
```

Comment:
i think we've already explained this back in chapter 5 when we first introduce the uow... is it one of the things brandon asked about?